numpy/core/tests/test_function_base.py

```diff
@@ -134,10 +134,11 @@ def test_boundaries_match_start_and_stop_exactly(self):
         assert_equal(y[0], start)
 
     def test_nan_midpoint(self):
-        y = geomspace(-3, 3, num=3)
-        assert_equal(y[0], -3.0)
-        assert_(isnan(y[1]))
-        assert_equal(y[2], 3.0)
+        with errstate(all='ignore'):
```

Comment:
I think you can be more precise here:
```suggestion
        with errstate(invalid='ignore'):
```

Reply:
done, thanks